contrib/initramfs/README.initramfs.markdown

```diff
@@ -2,7 +2,7 @@ DESCRIPTION
   These scripts is intended to be used with initramfs-tools, which is a similar
```

Comment:
There's another one here, s/scripts is intended/scripts are intended/.  While you're here can you fix this one too and take a look over the rest of the document.  Thanks!